src/Microsoft.Azure.WebJobs.ServiceBus/WebJobs.ServiceBus.csproj

```diff
@@ -2,12 +2,18 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Please move the common.props import to the top.

Reply:
Moved.